Update packaging-existing-software.md (#1031)
NixOS/nix.dev: source/tutorials/packaging-existing-software.md

This allows you to run `nix-build -A hello` to realize the derivation in `hello.nix`, similar to the current convention used in Nixpkgs.

:::{note}

> `callPackage` automatically passes attributes from `pkgs` to the given function, if they match attributes required by that function's argument attribute set. In this case, `callPackage` will supply `stdenv`, and `fetchzip` to the function defined in `hello.nix`.

Reviewer: ```suggestion
`callPackage` automatically passes attributes from `pkgs` to the given function, if they match attributes required by that function's argument attribute set.
In this case, `callPackage` will supply `stdenv`, and `fetchzip` to the function defined in `hello.nix`.
```

> The next chapter of this tutorial, [](./callpackage.md), goes into the details of how `callPackage` works.

Reviewer: ```suggestion
Refer to [](./callpackage.md) for more details on how this works.
```

:::{note}
`callPackage` automatically passes attributes from `pkgs` to the given function, if they match attributes required by that function's argument attribute set.
In this case, `callPackage` will supply `stdenv`, and `fetchzip` to the function defined in `hello.nix`.

> Refer to [](./callpackage.md) for more details on how this works.

Reviewer: ```suggestion
The tutorial [](./callpackage.md) goes into detail on how this works.
```
